slh-dsa/src/xmss.rs

```diff
@@ -0,0 +1,260 @@
+use hybrid_array::{Array, ArraySize};
+use typenum::Unsigned;
+
+use crate::wots::WotsSig;
+use crate::{address, wots::WotsParams};
+use crate::{PkSeed, SkSeed};
+use core::fmt::Debug;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct XMSSSig<P: XMSSParams> {
```

Comment:
Following the Rust RFC 430 naming conventions: https://rust-lang.github.io/api-guidelines/naming.html

The types/traits contained in this module should be prefixed `Xmss*` rather than `XMSS*`.

```suggestion
pub struct XmssSig<P: XmssParams> {
```